element_array_ephys/ephys_organoids.py

```diff
@@ -128,382 +129,185 @@ class AcquisitionSoftware(dj.Lookup):
-            else:
-                raise NotImplementedError(
-                    "Processing for neuropixels probe model"
-                    " {} not yet implemented".format(spikeglx_meta.probe_model)
-                )
+        subject_id = key["induction_id"]
+        data_dir = get_ephys_root_data_dir()[0] / subject_id
+        data_files = data_dir.glob(f"{subject_id}*.rhs")
+
+        # Loop through all the data files.
+        for file in sorted(list(data_files)):
+            # Load data
+            start_time = re.search(r".*_(\d{6}_\d{6})", file.stem).groups()[0]
+            start_time = np.datetime64(
+                datetime.strptime(start_time, "%y%m%d_%H%M%S")
+            )  # start time based on the file name
 
             self.insert1(
                 {
                     **key,
-                    **generate_electrode_config(probe_type, electrode_group_members),
-                    "acq_software": acq_software,
-                    "sampling_rate": spikeglx_meta.meta["imSampRate"],
-                    "recording_datetime": spikeglx_meta.recording_time,
-                    "recording_duration": (
-                        spikeglx_meta.recording_duration
-                        or spikeglx.retrieve_recording_duration(meta_filepath)
-                    ),
+                    "start_time": start_time,
+                    "file_name": file.relative_to(data_dir),
+                    "file_path": file,
                 }
             )
 
-            root_dir = find_root_directory(get_ephys_root_data_dir(), meta_filepath)
-            self.EphysFile.insert1(
-                {**key, "file_path": meta_filepath.relative_to(root_dir).as_posix()}
-            )
-        elif acq_software == "Open Ephys":
-            dataset = openephys.OpenEphys(session_dir)
-            for serial_number, probe_data in dataset.probes.items():
-                if str(serial_number) == inserted_probe_serial_number:
-                    break
-            else:
-                raise FileNotFoundError(
-                    "No Open Ephys data found for probe insertion: {}".format(key)
-                )
-
-            if not probe_data.ap_meta:
-                raise IOError(
-                    'No analog signals found - check "structure.oebin" file or "continuous" directory'
-                )
 
-            if probe_data.probe_model in supported_probe_types:
-                probe_type = probe_data.probe_model
-                electrode_query = probe.ProbeType.Electrode & {"probe_type": probe_type}
+@schema
+class EphysSessionInfo(dj.Imported):
+    definition = """
+    -> EphysSession
+    attribute_name      : varchar(32)
+    ---
+    attribute_blob=null : longblob
+    """
 
-                probe_electrodes = {
-                    key["electrode"]: key for key in electrode_query.fetch("KEY")
-                }
+    @property
+    def key_source(self):
+        return EphysSession - "session_type='spike_sorting'"
```

Comment:
Should we store this data regardless of whether we are running LFP analysis or spike sorting?

Reply:
If it's only for LFP, no,, but if there's going to be spike-sorting later on, I think this should be here.

Reply:
After discussion, we have decided to remove this update to the `key_source`.